src/Data/Connection/AbstractConnectionResolver.php

```diff
@@ -162,6 +157,22 @@ public function __construct( $source, array $args, AppContext $context, ResolveI
 		 */
 		$this->loader = $this->getLoader();
 
+		/**
+		 * Set the args for the resolver
+		 */
+		$this->args = $args;
+
+		/**
+		 *
+		 * Filters the GraphQL args before they are used in get_query_args().
+		 *
+		 * @param array                      $args                The GraphQL args passed to the resolver.
+		 * @param AbstractConnectionResolver $connection_resolver Instance of the ConnectionResolver
+		 *
+		 * @since @todo
+		 */
+		$this->args = apply_filters( 'graphql_connection_args', $this->get_args(), $this );
```

Comment:
I think I'm ok with this filter being added, but it might be a _bit_ naiive as it only targets connections, and not _any_ type of field. 

I was thinking it would make sense to allow filtering args on _any_ field, not just connections. 

Like described here: https://github.com/wp-graphql/wp-graphql/issues/965#issuecomment-539719493

I think it would be nice to be able to filter args _before_ they're used in resolution, but _also_ do something with the args _after_ resolution but _before_ the response is returned. 

Anyway, I think we can approach the bigger picture separately. This works for connections and has immediate benefits for working with connections.